CONTRIBUTING.md

```diff
@@ -0,0 +1,40 @@
+# Contributing to Open Liberty guides
```

Comment:
this file prob needs to be updated to reflect the space rover instead of the guides

Reply:
Done

Reply:
@jimmy1wu would you be able to approve if you're happy with the changes? TYIA 🙂